### PR TITLE
enable syntax highlight docker

### PR DESCRIPTION
**Pull request by gilacost**
## Intro

I just wanted to say hi 👋:
* I love shipit!
* I really love the BEAM! 
* I love ops!
* I don't like devops teams!
* I love nix!

## Why?

Yesterday I listened to the last [shipit 
podcast](https://changelog.com/shipit/51) and I loved it, as always. This is the first time  that I have realized that you use phoenix in your stack and that you might consider migrate to mix releases. So this morning I was trying to look around and check the dockerfile but there was no syntax highlight in github. 

## Commits

- chore(syntax-highlight): the dockerfile file type
- chore(syntax-highlight): replace references

## Questions

I  tried to buy a ship it T-shirt at the  changelog merchandise with no luck 😭. Where can I get one?

-----

Cheers all!

### CODE REVIEW

File: 2021/dagger/prod_image/main.cue [hunk 1]
```diff
@@ -13,7 +13,7 @@ prod_dockerfile:    dagger.#Input & {string}
 docker_host:        dagger.#Input & {string}
 dockerhub_username: dagger.#Input & {string}
 dockerhub_password: dagger.#Input & {dagger.#Secret}
-// ⚠️  Keep this in sync with ../docker/Dockerfile.production
+// ⚠️  Keep this in sync with ../docker//production.Dockerfile
```
Comment (gilacost) [inline 2021/dagger/prod_image/main.cue:16]:
```suggestion
// ⚠️  Keep this in sync with ../docker/production.Dockerfile
```

**Comment by gerhard** [comment]
> I just wanted to say hi 👋:

Hi! 👋

I cannot believe that it took me this long to find this PR! So glad that I'm doing this clean-up part of [Kaizen 7](https://changelog.com/shipit/70) prep.

> * I love shipit!

💙

> * I really love the BEAM!

All the way! 🐇

> * I love ops!
> * I don't like devops teams!

We are all devops! You build it, you run it.

> * I love nix!

https://twitter.com/gerhardlazu/status/1509537499567247360

> ## Why?
> Yesterday I listened to the last [shipit podcast](https://changelog.com/shipit/51) and I loved it, as always. This is the first time that I have realized that you use phoenix in your stack and that you might consider migrate to mix releases. So this morning I was trying to look around and check the dockerfile but there was no syntax highlight in github.

Thank you, I'll check this out over the next few days - this is the last comment for the day.

> ## Commits
> * chore(syntax-highlight): the dockerfile file type
> * chore(syntax-highlight): replace references

🙌

> ## Questions
> I tried to buy a ship it T-shirt at the changelog merchandise with no luck 😭. Where can I get one?

I would really like to get one myself! I'm thinking Christmas 2022 present to self. WDYT @jerodsanto @adamstac?

> Cheers all!

So glad to have you around 💪

**Comment by gilacost** [comment]
Let me know if I can help in any way 😄 🤗 🚀

**Comment by gerhard** [comment]
This will be in Kaizen 8: https://github.com/thechangelog/changelog.com/discussions/433 . Thank you very much for sticking with me 🙏

Kaizen t-shirts now available to order:

<a href="https://merch.changelog.com/products/kaizen" target="_blank">image</a>

**Pull request merged**